### PR TITLE
Update Sharp Aquos TV configuration to new style

### PR DESCRIPTION
**Pull request by dnguyen800**
**Description:**
Updates Sharp Aquos TV configuration variables to new style, as per #6385.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.aquostv.markdown [hunk 1]
```diff
@@ -27,14 +27,37 @@ media_player:
...
-- **password** (*Optional*): The password of the Sharp Aquos TV. Defaults to password.
-- **name** (*Optional*): The name you would like to give to the Sharp Aquos TV.
-- **power_on_enabled** (*Optional*): If you want to be able to turn on your TV. Defaults to False.
+{% configuration %}
+host:
+  description: The IP/Hostname of the Sharp Aquos TV, eg. `192.168.0.10`.
+  required: true
+  type: string
+port:
+  description: The port of the Sharp Aquos TV. Defaults to 10002.
```
Comment (fabaff) [inline source/_components/media_player.aquostv.markdown]: Please remove `Defaults ...`.

File: source/_components/media_player.aquostv.markdown [hunk 2]
```diff
@@ -27,14 +27,37 @@ media_player:
...
+  required: false
+  default: 10002
+  type: int
+username:
+  description: The username of the Sharp Aquos TV. Defaults to admin.
```
Comment (fabaff) [inline source/_components/media_player.aquostv.markdown]: Dito

File: source/_components/media_player.aquostv.markdown [hunk 3]
```diff
@@ -27,14 +27,37 @@ media_player:
...
+  required: false
+  default: admin
+  type: string
+password:
+  description: The password of the Sharp Aquos TV. Defaults to password.
```
Comment (fabaff) [inline source/_components/media_player.aquostv.markdown]: Dito

File: source/_components/media_player.aquostv.markdown [hunk 4]
```diff
@@ -27,14 +27,37 @@ media_player:
...
+  required: false
+  default: password
+  type: string
+name:
+  description: The name you would like to give to the Sharp Aquos TV.
+  required: false
+  type: string
+power_on_enabled:
+  description: If you want to be able to turn on your TV. Defaults to False.
```
Comment (fabaff) [inline source/_components/media_player.aquostv.markdown]: Dito

**Review by fabaff — CHANGES_REQUESTED** [review]

File: source/_components/media_player.aquostv.markdown [hunk 5]
```diff
@@ -27,14 +27,37 @@ media_player:
...
+  type: string
+name:
+  description: The name you would like to give to the Sharp Aquos TV.
+  required: false
+  type: string
+power_on_enabled:
+  description: If you want to be able to turn on your TV.
+  required: false
+  default: false
+  type: string
```
Comment (ludeeus) [inline source/_components/media_player.aquostv.markdown]: This should be `boolean`

Comment (dnguyen800) [reply source/_components/media_player.aquostv.markdown]: I thought so too, but I looked at the Home Assistant documentation and the allowed entries are: string, int, time, template or map. So I went with string.

https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

Comment (ludeeus) [reply source/_components/media_player.aquostv.markdown]: @dnguyen800 from that link:
```
type:: The type of the variable. Allowed entries: boolean, string, integer, float, time, template, device_class, icon or map/list (for a list of entries). For multiple possibilities use [string, integer]. If you use map/list then should define keys: (see the template sensor for an example). If you use boolean, then default: must be defined.
```
:)

Comment (dnguyen800) [reply source/_components/media_player.aquostv.markdown]: haha, that's funny! I am 100% sure it did not show boolean, float, device_class, or icon when I viewed that site yesterday. Anyways, I'll change it to boolean right now.

**Review by fabaff — APPROVED** [review]

**Pull request merged**